automation_tools/baseimage.py

```diff
@@ -57,6 +57,7 @@ def create_baseimage(os_url, image=None, auth_keys_url=None, dns_server=None, di
 
     put('misc/base_image_creation/ks_rhel{}_template'.format(os_ver), 'ks.cfg')
     run('sed -i "s|OS_URL|{}|g" ks.cfg'.format(os_url))
+    run('sed -i "s|AS_URL|{}|g" ks.cfg'.format(os_url.replace('BaseOS', 'AppStream')))
```

Comment:
This should be run for just rhel8 right?

Reply:
Yes for RHEL8

For RHEL7 it's safe since ks template has no "AS_URL" (sed replace nothing)

Reply:
I changed my mind and incorporated change into PR